collections/_infographics/energy/electricity-per-capita-world.md

```diff
@@ -14,4 +14,29 @@ data-orig:
   - [ "UN (population)", "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/EXCEL_FILES/1_Population/WPP2019_POP_F01_1_TOTAL_POPULATION_BOTH_SEXES.xlsx" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/elektrina-na-osobu-svet" %}
+## How to read this graph
+
+* The graph shows the **worldwide development of electricity produced per person by individual sources in the years 2000-2019**, as well as a comparison with selected world regions. Values are given in **<glossary id="w">kWh</glossary> of electricity produced per person per year**.
```

Comment:
change *a comparison with selected* to *a comparison of selected*.

Reply:
Updated